phpstan.neon.dist

```diff
@@ -2,10 +2,12 @@ parameters:
     level: 7
     reportUnmatchedIgnoredErrors: false
     inferPrivatePropertyTypeFromConstructor: true
+    checkMissingIterableValueType: false
     symfony:
         constant_hassers: false
-        # the placeholder "%ShopwareHashedCacheDir%" will be replaced on execution by bin/phpstan-config-generator.php script
```

Comment:
why did you remove this? 

if it is not needed anymore, the `phpstan-config-generator.php` is also obsolete :wink:

Reply:
Yes, I think this can be removed (see also the comment in the pull request itself), since all Shopware files should be properly autoloaded as they are installed via composer in the Docker image. I would then also remove this file.

I am currently not sure if the container is still needed for the phpstan analysis or if this is obsolete, see also the following issue:
https://github.com/aragon999/phpstan-shopware-docker/issues/1

Reply:
https://github.com/FriendsOfShopware/FroshPlatformThumbnailProcessor/pull/4/commits/4a5e872be95f876310c079079c9f14c0531b2292 :-)